test/DomainRegistry.test.js

```diff
@@ -0,0 +1,87 @@
+const { expect } = require("chai");
+
+describe("DomainRegistry", function () {
+  let domainRegistry;
+  let owner;
+
+  beforeEach(async function () {
+    const hre = require("hardhat");
+    const DomainRegistry = await hre.ethers.getContractFactory("DomainRegistry");
+    domainRegistry = await DomainRegistry.deploy();
+    [owner] = await hre.ethers.getSigners();
+  });
+
+  it("Should register a new .com domain", async function () {
+    const domainForThisTest = "com";
+    const tx = await domainRegistry.connect(owner).registerDomain(domainForThisTest, {
+        value: hre.ethers.parseEther("1.0"),
+    });
+    await tx.wait();
```

Comment:
FYI Waiting for tx receipt is not mandatory for most cases in the tests

Reply:
fxd